index.html

```diff
@@ -1,89 +1,294 @@
+                    class="last fa fa-heart"
+                    style="font-size: 20px"
+                  ></i>
+                </h4>
+              </div>
+            </div>
+            <div class="prat col-md-6 mt-4 mt-md-0">
+              <div class="icon-box">
+                <i class="icofont-chart-bar-graph"></i>
+                <h4>
+                  <i class="fa fa-times-circle" style="font-size: 40px"></i
+                  >&nbsp Avoid Stress<i
+                    class="last fa fa-heart"
+                    style="font-size: 20px"
+                  ></i>
+                </h4>
+              </div>
+            </div>
+            <div class="prat col-md-6 mt-4 mt-md-0">
+              <div class="icon-box">
+                <h4>
+                  <i class="fa fa-bar-chart" style="font-size: 40px"></i>&nbsp
+                  Eat Healthy food<i
+                    class="last fa fa-heart"
+                    style="font-size: 20px"
+                  ></i>
+                </h4>
+              </div>
+            </div>
+            <div class="prat col-md-6 mt-4 mt-md-0">
+              <div class="icon-box">
+                <i class="icofont-settings"></i>
+                <h4>
+                  <i class="fa fa-check-circle" style="font-size: 40px"></i
+                  >&nbsp Get enough sleep<i
+                    class="last fa fa-heart"
+                    style="font-size: 20px"
+                  ></i>
+                </h4>
+              </div>
+            </div>
+            <div class="prat col-md-6 mt-4 mt-md-0">
+              <div class="icon-box">
+                <i class="icofont-settings"></i>
+                <h4>
+                  <i class="fa fa-check-circle" style="font-size: 40px"></i
+                  >&nbsp Excercise or physical activity<i
+                    class="last fa fa-heart"
+                    style="font-size: 20px"
+                  ></i>
+                </h4>
+              </div>
+            </div>
+          </div>
         </div>
-    <br>
-  
+      </div>
+    </div>
 
-      <br><br><br>
-      
+    <!-- footer -->
+    <div class="footer">
+      <div class="container">
+        <div class="row">
+          <div class="footer-col-1">
+            <h3>Download Our App</h3>
+            <p>Download App for Android and ios mobile phone</p>
+            <div class="app-logo">
+              <img src="static/img/play-store.png" alt="" />
+              <img src="static/img/app-store.png" alt="" />
+            </div>
+          </div>
+          <div class="footer-col-2">
+            <h1>Vaccine Tracker</h1>
+            <p>Our goal is to help the world free from the ongoing pandemic.</p>
+          </div>
+          <div class="footer-col-3">
+            <h3>Useful Links</h3>
+            <ul>
+              <li>Coupons</li>
+              <li>Blog Post</li>
+              <li>Return Policy</li>
+              <li>Join Affiliate</li>
```

Comment:
Also @Biswa-bob few extra part you have added in the footer which was not needed. For now, we just wanted the copyright symbol with text `Made with ❤️ for the people of India.`

Reply:
sure brother